src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/205369181-31553bce-e674-4715-a54f-55fdcdb72a5c.png)
Please, follow the checklist
```suggestion
    private List<Long> ticketIds;
```